EU4ToVic3/Source/V3World/EconomyManager/Demand/MarketJobs.h

```diff
@@ -0,0 +1,67 @@
+#ifndef V3_MARKET_JOBS_H
+#define V3_MARKET_JOBS_H
+
+#include "PoliticalManager/Country/Country.h"
+#include "PopManager/Pops/PopType.h"
+
+namespace V3
+{
+class MarketJobs
+{
+  public:
+	MarketJobs() = default;
+	explicit MarketJobs(const std::vector<std::pair<std::string, int>>& manorHouseRoster);
+
+	double createJobs(const std::map<std::string, double>& rgoUnitEmployment,
+		 const std::map<std::string, double>& subsistenceUnitEmployment,
+		 int levels,
+		 double defaultRatio,
+		 double womenJobRate,
+		 const std::map<std::string, double>& estimatedOwnerships,
+		 const std::map<std::string, std::map<std::string, int>>& ownershipEmployments,
+		 const std::map<std::string, PopType>& popTypes,
+		 const std::shared_ptr<SubState>& subState);
+	double createSubsistence(const std::map<std::string, double>& subsistenceUnitEmployment,
+		 double defaultRatio,
+		 double womenJobRate,
+		 int arableLand,
+		 const std::map<std::string, PopType>& popTypes,
+		 const std::shared_ptr<SubState>& subState);
+
+  private:
+	// For a given popType, what % of that popType have jobs?
+	static double calculateWorkerDependencyRatio(const PopType& popType, double defaultRatio, double womenJobRate);
+	// How many people do x number of jobs support?
+	static double getWorkersPlusDependents(double amountOfJobs, const PopType& popType, double defaultRatio, double womenJobRate);
+
+	// Returns the amount of jobs with no unemployed available.
+	static double hireFromUnemployed(double amount, const std::shared_ptr<SubState>& subState);
+	// Returns the level of subsistence buildings downsized.
```

Comment:
```suggestion
	// For a given popType, what % of that popType have jobs?
	static double calculateWorkerDependencyRatio(const PopType& popType, double defaultRatio, double womenJobRate);
	// How many people do x number of jobs support?
	static double getWorkersPlusDependents(double amountOfJobs, const PopType& popType, double defaultRatio, double womenJobRate);

	// Returns the amount of jobs with no unemployed available.
	static double hireFromUnemployed(double amount, const std::shared_ptr<SubState>& subState);
	// Returns the level of subsistence buildings downsized.
```

Since these aren't used outside the class, it would be better to put them in an anonymous namespace at the top of the .cpp file and not expose them in the interface.